### DISCUSSION TITLE
duckdns

### DISCUSSION THREAD

**Discussion by forziere** [indomus/forum#2145] [HUB: Home Assistant]
ciao a tutti,
ho provato un sacco di volte a mettere qs addon ma mi dice sempre errore. io ho un router vodafone e ho messo
![image](https://user-images.githubusercontent.com/73720066/206024821-44e5df94-2c2a-4f23-8f1b-ae117e2e5f73.png)
sia per la 443 che per la 8123 ma non funziona errore dopo aver messo il mio numero di token e dominio.. 
mi dareste una mano??

**Comment by calz64** [comment]
Non sono un grande esperto, quindi ti posso dire che con Fritzbox (fra parentesi tutto un altro mondo dai router della cippa che ti danno i gestori facendoli pagare un sacco, chiusa parentesi) mi chiede di impostare anche la porta esterna sull'ip pubblico, può essere quello?

**Reply by snakuzzo** [reply]
La considerazione sul fritzbox non c'entra niente con un semplice porta forwarding.

**Reply by snakuzzo** [reply]
@forziere ma sei sicuro di averlo configurato correttamente il port forwarding?
HA in locale su che porta risponde?

Se per esempio HA in locale risponde sulla 8123 e da fuori invece vuoi raggiungerlo sulla 443, devi configurare il port forwarding con porta esterna 443 ed interna 8123

**Comment by forziere** [comment]
Grazie Daniele.
Mi compare un errore errore tipo ...manca option  dominio quando provo ad
avviare duckdns

Il Gio 8 Dic 2022, 11:16 daniele ***@***.***> ha scritto:

**Reply by snakuzzo** [reply]
???
Ma hai letto la documentazione dell'addon?

Altra cosa...
Non fare reply dalle mail e rispondi sotto l'ultimo messaggio, non aprire nuovi thread come hai appena fatto.
Grazie
